### PR TITLE
Ensure dev-tools uses bookworm

### PR DESCRIPTION
**Pull request by dato**
Fixes: #3190.

### CODE REVIEW

**Comment by dato** [comment]
This is just a precaution, we could as well close the bug with the issue itself acting as documentation. Should be very rare. But, if you prefer, we can apply this instead.

File: dev-tools/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.9
+FROM python:3.9-bookworm
```
Comment (mouse-reeve) [inline dev-tools/Dockerfile:1]: is there a python version coincidentally called "bookwyrm" or am I misunderstanding this??

**Review by mouse-reeve — COMMENTED** [review]
I think it's a good idea to fix it here, I'd rather things Just Work rather than needing to be documented wherever possible

File: dev-tools/Dockerfile (hunk 1 above)
Comment (dato) [reply dev-tools/Dockerfile:1]: Debian 12 is code-named "bookworm". :)

So these are the Python images based on Debian 12.

Comment (mouse-reeve) [reply dev-tools/Dockerfile:1]: wild!

**Pull request merged**